packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -15,6 +17,13 @@ import '../../table_data.dart';
 import '../../ui/flutter/service_extension_widgets.dart';
 import '../logging_controller.dart';
 
+// TODO(devoncarew): Show rows starting from the top (and have them grow down).
+// TODO(devoncarew): We should keep new items visible (if the last item was
```

Comment:
I thought the existing implementation already keeps new items visible if the last item was already visible.

Reply:
It does, by reversing scrolling and table entry.

This isn't ideal though:
- the rows start out drawing on the bottom for empty tables - this looks odd, esp. when the table headers are ~800 px higher up
- on new items, the table always scrolls, even if you're looking at items higher up (and have manually adjusted the scroll position). It's really hard to click on items if you have an app which logs regularly

I think we want to re-work things here; remove the `reverse` code, auto-scroll if the scroll position is at the bottom, and un-dock from auto-scrolling if the user has manually scrolled the last back up a bit.